docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3298_1.md

```diff
@@ -0,0 +1,37 @@
+# Restart jobs in other regions shouldn't work if Allow reschedule option is disabled
+
+Test verifies that restart jobs in other regions shouldn't work if _Allow reschedule to different region in case of insufficient capacity_ option for tool is disabled.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **PREFERENCES** tab
+4. Find the **`run.reschedule.enabled`** preference
+5. Check *Enabled* checkbox for `run.reschedule.enabled` preference if needed
+6. Click **Save** button if needed
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select test tool | |
+| 4 | Open **Settings** tab | |
+| 5 | Expand **EXECUTION ENVIRONMENT** | |
+| 6 | Select ***Disabled*** option for _Allow reschedule to different region in case of insufficient capacity_ | |
+| 7 | Click **Save** button if needed | |
+| 8 | At the tool page, hover over the **Run v** button | |
+| 9 | Click the **Custom settings** button in the list | |
+| 10 | Expand the **Exec environment** section | |
+| 11 | Select the **Node type** from the prerequisites | |
+| 12 | Select `region1` value (same as in step 4 of the preparations of case [3122_1](3122_1.md)) for **Cloud Region** | |
+| 13 | Expand the **Advanced** section | |
+| 14 | Select `On-demand` value for **Price type** | |
+| 15 | Launch the run | |
+| 16 | At the **Runs** page, click the just-launched run | |
+| 17 | Wait until ***RestartPipelineRun*** task appears | <li> Run is stopped <li> Message `Server failure reason: Insufficient instance capacity.` appears under the run title |
+| 18 | Click ***RestartPipelineRun*** task | Records <br> `Run '<runID>' failed to start.` <br> `Cannot restart run '<runID>': run is not configured for restart.`, where `<runID>` is ID run launched at step 15   <br> are shown in the log |
```

Comment:
Add the section "**After**" with the content:
- Restore the value of the option _Allow reschedule to different region in case of insufficient capacity_ that was in place before step 6
- Restore the value of the `run.reschedule.enabled` preference to the previous value saved at step 12 of the preparations of [3122_1](3122_1.md) case